op-interop-mon/monitor/finder_test.go

```diff
@@ -56,7 +56,7 @@ func (m *mockClient) FetchReceiptsByNumber(ctx context.Context, number uint64) (
 	}
 }
 
-func mockReceiptsToCases(receipts []*types.Receipt) []*Job {
+func mockReceiptsToCases(receipts []*types.Receipt, executingChain eth.ChainID) []*Job {
```

Comment:
"Cases" was a very old name for Job that seems to have survived in this mock. Can we change these references to Job while we're here please?

Reply:
[69136a4](https://github.com/ethereum-optimism/optimism/pull/16494/commits/69136a426d22f9f9ff2fb4d1135a8461a54601a0)